phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -443,6 +448,11 @@ public ConnectionQueryServicesImpl(QueryServices services, ConnectionInfo connec
         for (Entry<String,String> entry : connectionInfo.asProps()) {
             config.set(entry.getKey(), entry.getValue());
         }
+        if (connectionInfo.getPrincipal() != null) {
+            config.set(QUERY_SERVICES_NAME, connectionInfo.getPrincipal());
+        }
+        LOGGER.info(String.format("CQS initialized with connection profile : %s",
```

Comment:
Maybe the log should read "CQS initialized with query service name"

Reply:
done